descriptors/ModuleDescriptor-template.json

```diff
@@ -408,6 +459,43 @@
         "invoice.acquisitions-units-assignments.assignments.item.delete"
       ]
     },
+    {
+      "permissionName": "voucher.acquisitions-units-assignments.assignments.collection.get",
```

Comment:
I think the `.assignments.` part is superfluous and not needed

Reply:
applies to all of these...

Reply:
If we remove `.assignments` then it will be inconsistent with `invoice.acquisitions-units-assignments.assignments.` Do you want me to change for `invoice` too?

Reply:
oops, how'd that get through... yeah let's change the invoice ones too

Reply:
Done